Description: gcc: format not a string literal and no format arguments
Author: Jari Aalto <jari.aalto@cante.net>
Last-update: 2012-02-08

From 8fd99c3c6a6fdf7bb3dc7a2cd786e8a77011c879 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Wed, 8 Feb 2012 06:59:36 -0500
Subject: [PATCH] gcc: format not a string literal and no format arguments
Organization: Private
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit
Signed-off-by: Jari Aalto <jari.aalto@cante.net>

Reviewed-by: Paulo Roberto Alves de Oliveira (aka kretcheu) <kretcheu@gmail.com>
Last-update: 2015-12-08
---
 src/dialog.c   |    4 ++--
 src/gtkprint.c |    2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/dialog.c b/src/dialog.c
index 14b69d7..06c0f21 100644
--- a/src/dialog.c
+++ b/src/dialog.c
@@ -36,7 +36,7 @@ void run_dialog_message(GtkWidget *window,
 		GTK_DIALOG_DESTROY_WITH_PARENT,
 		type,
 		GTK_BUTTONS_NONE,
-		str);
+                (const gchar*) str, "");
 	gtk_window_set_resizable(GTK_WINDOW(dialog), FALSE);
 	gtk_dialog_add_buttons(GTK_DIALOG(dialog),
 		GTK_STOCK_OK, GTK_RESPONSE_CANCEL, NULL);
@@ -61,7 +61,7 @@ GtkWidget *create_dialog_message_question(GtkWidget *window, gchar *message, ...
 		GTK_DIALOG_DESTROY_WITH_PARENT,
 		GTK_MESSAGE_QUESTION,
 		GTK_BUTTONS_NONE,
-		str);
+		(const gchar*) str, "");
 	gtk_window_set_resizable(GTK_WINDOW(dialog), FALSE);
 	gtk_dialog_add_buttons(GTK_DIALOG(dialog),
 		GTK_STOCK_NO, GTK_RESPONSE_NO,
diff --git a/src/gtkprint.c b/src/gtkprint.c
index 3f39384..476a2be 100644
--- a/src/gtkprint.c
+++ b/src/gtkprint.c
@@ -165,7 +165,7 @@ static void create_error_dialog(GtkTextView *text_view, gchar *message)
 		GTK_DIALOG_DESTROY_WITH_PARENT,
 		GTK_MESSAGE_ERROR,
 		GTK_BUTTONS_NONE,
-		message);
+		(const gchar*) message, "");
 	gtk_window_set_resizable(GTK_WINDOW(dialog), FALSE);
 	gtk_dialog_add_buttons(GTK_DIALOG(dialog),
 		GTK_STOCK_OK, GTK_RESPONSE_CANCEL, NULL);
-- 
1.7.9

